contentgrid-spring-data-support/src/main/java/com/contentgrid/data/support/auditing/v1/AuditMetadata.java

```diff
@@ -0,0 +1,35 @@
+package com.contentgrid.data.support.auditing.v1;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.persistence.Embeddable;
+import java.time.Instant;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.annotation.LastModifiedDate;
+
+@Embeddable
+@NoArgsConstructor
+@Getter
+@Setter
+public class AuditMetadata {
+
+    @CreatedBy
+    @JsonProperty(value = "created_by")
+    private UserMetadata createdBy;
+
+    @CreatedDate
+    @JsonProperty(value = "created_date")
+    private Instant createdDate;
+
+    @LastModifiedBy
+    @JsonProperty(value = "last_modified_by")
+    private UserMetadata lastModifiedBy;
+
+    @LastModifiedDate
+    @JsonProperty(value = "last_modified_date")
+    private Instant lastModifiedDate;
```

Comment:
I think these fields should also be marked as json read only?

Reply:
no longer necesssary when top-level `AuditMetadata` field is marked as read-only